app/_includes/md/kgo/konnect-entities-prerequisites.md

```diff
@@ -0,0 +1,133 @@
+{% unless include.disable_accordian %}
+<details class="custom" markdown="1">
+<summary>
+<blockquote class="note">
+  <p style="cursor: pointer">Before you create any Konnect entity, make sure you've <u>installed {{site.kgo_product_name}} and created a valid KonnectAPIAuthConfiguration{% if include.with-control-plane %} and KonnectGatewayControlPlane{% endif %}</u> in your cluster.</p>
+</blockquote>
+</summary>
+
+## Prerequisites
+{% endunless %}
+
+{% include md/kgo/prerequisites.md disable_accordian=true version=page.version release=page.release kconf-crds=true %}
+
+### Create an access token in Konnect
+
+You may create either a Personal Access Token (PAT) or a Service Account Token (SAT) in Konnect. Please refer to the
+[Konnect authentication documentation](/konnect/api/#authentication) for more information. You will need this token
+to create a `KonnectAPIAuthConfiguration` object that will be used by the {{site.kgo_product_name}} to authenticate
+with Konnect APIs.
+
+### Create a Konnect API Auth Configuration
```

Comment:
```suggestion
### Create a {{site.konnect_product_name}} API auth configuration
```

Reply:
I'm not sure about lower casing this. I referred to [Writing for Kong](https://konghq.atlassian.net/wiki/spaces/MT/pages/3018883164/Writing+for+Kong+Style+Guide) guide and it says:
> Use [title case](https://capitalizemytitle.com/style/Chicago/) for titles of pieces and calls to action. Use sentence case for all other headers.

This header is a call to action, so I'd keep it capitalized, WDYT?

Reply:
Ah, the marketing style guide is different than the [docs style guide. ](https://docs.konghq.com/contributing/style-guide/#headings-and-page-titles). We specifically use sentence case for all headers.

Reply:
Fixed in 3100cd5